packages/flutter_tools/lib/src/macos/build_macos.dart

```diff
@@ -18,6 +18,7 @@ Future<void> buildMacOS(FlutterProject flutterProject, BuildInfo buildInfo) asyn
     flutterProject.macos.buildScript.path,
     Cache.flutterRoot,
     buildInfo?.isDebug == true ? 'debug' : 'release',
+    buildInfo?.trackWidgetCreation == true ? 'true' : 'false',
```

Comment:
Let's add a bit of context to make the scripts more self-documenting (since they are already using magic positional elements); how about 'track-widget-creation' and 'no-track-widget-creation' or something?

Reply:
Done